Gemfile

```diff
@@ -10,9 +10,11 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gemspec
 
+# NOTE: we no longer publish internal ALI gems to rubygems.org, so we use GitHub
```

Comment:
So sad!  Not that I'm using it, nor is anybody else. :(. Just lurking.  Do what you need to do with all blessing, and best wishes!